src/Section1/Section1.js

```diff
@@ -1,15 +1,14 @@
-import { Container, Title, Image } from './style';
-
-
+import { Container, Title, Image, HiddenOnMobile } from "./style";
 
 export const Section1 = () => {
-    return(
-        <Container>
-            <Title>Learning how to code</Title>
-            <Image />
-            
-        </Container>
-    )
-}
-
-export default Section1;
\ No newline at end of file
+  return (
+    <Container>
+      <Title>
+        Learning <HiddenOnMobile>how</HiddenOnMobile> to code
+      </Title>
+      <Image />
+    </Container>
+  );
+};
+
+export default Section1;
```

Comment:
Since you are naming importing this component (ie. `import { Section1 } from '...'`) you don't need to default export.
You could just remove this default export.

Default exports would be necessary if you would want to import it like: `import Section from './Section1'` It would get the default exported component, no matter what it is named.